integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/CoupleMBusDevice.feature

```diff
@@ -423,3 +423,157 @@ Feature: SmartMetering Installation - Couple M-Bus Device
       | Code    |                                     222 |
       | Message | MBUS_DEVICE_NOT_MOVED_TO_ANOTHER_EMETER |
     And the mbus device "TESTG101205673117" is not coupled to the device "TEST1024000000001"
+
+  Scenario: Couple G-meter to an E-meter, without anything connected
```

Comment:
Mooie duidelijke scenario's. Wel veel dubbel. Is er iets te doen met een scenario outline, met force true/false en coupledGateway null/TEST1024000000001/TEST1024000000002 ?